README.md

```diff
@@ -82,6 +82,15 @@ RIOT provides features including, but not limited to:
 * UWB
 * Bluetooth (BLE) via [NimBLE](https://github.com/apache/mynewt-nimble)
 
+### Related Projects
+
+- [**Ariel OS**](https://github.com/ariel-os/ariel-os) is an offspring of RIOT written
+in Rust following RIOT's goals and [vision](https://doc.riot-os.org/vision.html).
```

Comment:
```suggestion
- [**Ariel OS**](https://github.com/ariel-os/ariel-os) is young IoT OS with a
  focus on security and memory safety that is inspired by RIOT and follows the
  same goals and [vision](https://doc.riot-os.org/vision.html).
```

How about this wording It avoids the offspring/spinoff wording, that seem to be the main point of friction.